packages/round-manager/src/features/round/ViewRoundSettings.tsx

```diff
@@ -1827,16 +1830,19 @@ function Funding(props: {
         </div>
       </div>
       <div>
```

Comment:
This still prints an empty `div`, might be cleaner to set the `className` of the `div` to hidden if "no" is selected?

Same below for the other banner.

Reply:
I wrapped the entire div instead so we should expect the same behavior.